src/components/record/record-page-action-buttons.tsx

```diff
@@ -0,0 +1,58 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_record.scss';
+import Button from 'react-bootstrap/Button';
+import ChaiseTooltip from '@isrd-isi-edu/chaise/src/components/tooltip';
+import Export from '@isrd-isi-edu/chaise/src/components/export';
+
+/**
+ * Page Action Types can be create, delete, copy, and edit
+ */
+export enum PAGE_ACTION_TYPES {
+  SHOW_EMPTY,
+  SHARE_CITE,
+}
+
+export type RecordPageActionButtonsProps = {
+  onAction: (type: PAGE_ACTION_TYPES, event: any) => void;
+};
+
+/**
+ * Returns Related Section of the record page.
+ */
+const RecordPageActionButtons = ({
+  onAction,
+}: RecordPageActionButtonsProps): JSX.Element => {
+  return (
+    <>
+      <ChaiseTooltip
+        placement='bottom-start'
+        tooltip='Click here to show empty related sections too.'
+      >
+        <Button className='chaise-btn chaise-btn-primary' onClick={(event: any) => onAction(PAGE_ACTION_TYPES.SHOW_EMPTY, event)}>
+          <span className='chaise-btn-icon fa fa-th-list'></span>
+          Show empty sections
+        </Button>
+      </ChaiseTooltip>
+      <ChaiseTooltip
+        placement='bottom-start'
+        tooltip='Click here to show an export format'
+      >
+        <Export
+          reference={null}
+          // disabled={isLoading || !page || page.length === 0}
```

Comment:
When you moved this to `record.tsx`, you should pass proper values to it and make sure it's working properly in context of record as well. We don't have a `isLoading` for now, so let's just set the disabled state based on `page` (and add a TODO saying that it should also be disabled while loading).